setup.py

```diff
@@ -37,6 +37,12 @@
     ('/opt/intelmq/var/lib/bots/file-output/',
      [],
      ),
+    ('/usr/bin',
```

Comment:
Just noticed that this affects all installations, not only the packaged ones. These scripts should go to the [`console_scripts`](https://github.com/certtools/intelmq/blob/master/setup.py#L96), not "hard"copied to `/usr/bin`. They may be installed there, if the path for binaries is set to `/usr/bin`, which is the case for setuptools used for packaging, but not for locally installed instances.